### ISSUE TITLE
UIA in conhost: incorrect word boundaries when text is selected

### ISSUE THREAD

**Issue by codeofdusk** [microsoft/terminal#11809]
### Windows Terminal version

Version of OpenConsole included with the wt 1.12 release.

### Windows build number

10.0.22000.348

### Other Software

NVDA master (2022.1) with UIA.

### Steps to reproduce

Paste the following text into a conhost. On my system, this line wraps at the word "upgrade":

`tree.IncompatibleFormatError: This file is too new for the currently running version of Treemendous (0.0.0). Please upgrade to Treemendous 42.0.0 or later.`

Then:
* Select a word, such as by moving to the top of the console and searching for the word "please" with CTRL+f.
* navigate forward in the window by word.

### Expected Behavior

word boundaries are correctly calculated: after the wrapping, NVDA says "upgrade", "to", "Treemendous", etc.

### Actual Behavior

NVDA reads from the start of the line up to some arbitrary point, can't tell what's going on???

**Comment by codeofdusk** [comment]
This *does not* repro in wt.

CC @carlos-zamora, @DHowett.

**Comment by codeofdusk** [comment]
False alarm! Looks like a Windows update replaced my `FORMATTED` console with an `IMPROVED` one for some reason. This works as intended in `FORMATTED` conhost!

**Comment by codeofdusk** [comment]
Confirmed repro of this issue in `FORMATTED` conhost, but looks like this is the only remaining word boundary bug!

**Comment by codeofdusk** [comment]
On further testing, it is important that the selected text *must* be on a line that wraps, and the selection must be somewhere on that line. NVDA+f9/f10 also works for selection here (not just CTRL+f), so I presume the mouse would as well. Also CC @DHowett.

**Comment by alterseries83** [comment]
Same here.

**Comment by zadjii-msft** [comment]
This might be related to #5800, or some of the weird line wrapping bugs that I've at least been linking up to that one. I'm thinking particularly the #6901, #10833, #11739 subset.

**Comment by codeofdusk** [comment]
@zadjii-msft This bug explicitly doesn't repro in wt (conhost *only*). Please re-label.

**Comment by zadjii-msft** [comment]
.... pretty sure I only added the conhost label...
![image](https://user-images.githubusercontent.com/18356694/144871416-1938d012-2bb9-49b9-b420-3e62a1e459a8.png)
![image](https://user-images.githubusercontent.com/18356694/144871526-f88dbae4-f28f-482d-b6a2-ec08ec57f67d.png)

**Comment by codeofdusk** [comment]
Can't see image but it says you added it to the terminal v2.0 milestone. Should be in 22H1.

**Comment by zadjii-msft** [comment]
Ah. I've been kinda re-thinking how we track milestones for our issues recently, since the Windows-specific ones don't totally make sense and have been pretty loosely tracked, and there's not really a difference between the Terminal and Console backlogs anymore. I suppose for the time being, I'll slap that into the 22H1 milestone, but we might be shaking those up in the future, FYI.

**Comment by codeofdusk** [comment]
FYI this repros in the SV2 inbox console. CC @carlos-zamora, @DHowett.

**Comment by zadjii-msft** [comment]
@codeofdusk Yep, looks like this one didn't make it in unfortunately. Hopefully we'll get a chance for the next release.
